docs/src/tutorials/xla_compilation.md

```diff
@@ -0,0 +1,82 @@
+# Compiling NeuralOperators.jl using Reactant.jl
+
+```@example xla_compilation
+using NeuralOperators, Lux, Random, Enzyme, Reactant
+
+function sumabs2first(model, ps, st, (u, y))
+    z, _ = model((u, y), ps, st)
+    return sum(abs2, z)
+end
+
+if "gpu" in keys(Reactant.XLA.backends)
```

Comment:
GPU is automatically set if availalbe

Reply:
https://github.com/EnzymeAD/Reactant.jl/pull/272

Reply:
welll....that's what I thought what was happening (and remember at one point havng that), but I guess that got messed up at one point....